src/main/java/Duke.java

```diff
@@ -1,10 +1,287 @@
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String div = "____________________________________________________________\n";
+
+        class Task {
```

Comment:
Should classes be declared inside methods?

Reply:
I think it is better for you to separate your classes into different files, especially when such class declarations are taking up a significant amount of space in your file.